src/profile-logic/call-node-info.js

```diff
@@ -274,51 +270,298 @@ export class CallNodeInfoImpl implements CallNodeInfo {
+  let previousRootSuffixOrderIndexRangeEnd = 0;
+  for (let funcIndex = 0; funcIndex < funcCount; funcIndex++) {
+    const callNodesuffixOrderIndexRangeStart =
+      previousRootSuffixOrderIndexRangeEnd;
+    const callNodesuffixOrderIndexRangeEnd =
+      rootSuffixOrderIndexRangeEndCol[funcIndex];
+    previousRootSuffixOrderIndexRangeEnd = callNodesuffixOrderIndexRangeEnd;
+    if (
+      callNodesuffixOrderIndexRangeStart === callNodesuffixOrderIndexRangeEnd
+    ) {
+      sourceFramesInlinedIntoSymbol[funcIndex] = null;
+      // Leave the remaining columns at zero for this root.
+      continue;
+    }
+
+    // Fill the remaining fields with the conflict-resolved versions of the values
+    // in the non-inverted call node table.
+    const firstNonInvertedCallNodeIndex =
+      suffixOrderedCallNodes[callNodesuffixOrderIndexRangeStart];
+    let resolvedCategory =
+      callNodeTable.category[firstNonInvertedCallNodeIndex];
+    let resolvedSubcategory =
+      callNodeTable.subcategory[firstNonInvertedCallNodeIndex];
+    const resolvedInnerWindowID =
+      callNodeTable.innerWindowID[firstNonInvertedCallNodeIndex];
+    let resolvedSourceFramesInlinedIntoSymbol =
+      callNodeTable.sourceFramesInlinedIntoSymbol[
+        firstNonInvertedCallNodeIndex
+      ];
+
+    // Resolve conflicts in the same way as for the non-inverted call node table.
+    for (
+      let orderingIndex = callNodesuffixOrderIndexRangeStart + 1;
+      orderingIndex < callNodesuffixOrderIndexRangeEnd;
+      orderingIndex++
+    ) {
+      const currentNonInvertedCallNodeIndex =
+        suffixOrderedCallNodes[orderingIndex];
+      // Resolve category conflicts, by resetting a conflicting subcategory or
+      // category to the default category.
+      if (
+        resolvedCategory !==
+        callNodeTable.category[currentNonInvertedCallNodeIndex]
+      ) {
+        // Conflicting origin stack categories -> default category + subcategory.
+        resolvedCategory = defaultCategory;
+        resolvedSubcategory = 0;
+      } else if (
+        resolvedSubcategory !==
+        callNodeTable.subcategory[currentNonInvertedCallNodeIndex]
+      ) {
+        // Conflicting origin stack subcategories -> "Other" subcategory.
+        resolvedSubcategory = 0;
+      }
+
+      // Resolve "inlined into" conflicts. This can happen if you have two
+      // function calls A -> B where only one of the B calls is inlined, or
+      // if you use call tree transforms in such a way that a function B which
+      // was inlined into two different callers (A -> B, C -> B) gets collapsed
+      // into one call node.
+      if (
+        resolvedSourceFramesInlinedIntoSymbol !==
+        callNodeTable.sourceFramesInlinedIntoSymbol[
+          currentNonInvertedCallNodeIndex
+        ]
+      ) {
+        // Conflicting inlining: -1.
+        resolvedSourceFramesInlinedIntoSymbol = -1;
+      }
+
+      // FIXME: Resolve conflicts of InnerWindowID
```

Comment:
Yes, I intentionally left it out because it's also missing for the non-inverted case, when creating the call node table from the stack table. And I don't know what should be done here, or how innerWindowIDs are used in the call node table at all. If we don't need innerWindowIDs in the call node table, my preferred fix would be to remove the column.